docs/reference/edot-collector/components/k8sclusterreceiver.md

```diff
@@ -0,0 +1,204 @@
+---
+navigation_title: Kubernetes Cluster receiver
+description: Collects Kubernetes cluster-level metrics and entity events for Elastic Observability through the EDOT Collector.
+applies_to:
+  stack:
+  serverless:
+    observability:
+  product:
+    edot_collector:
+products:
+  - id: cloud-serverless
+  - id: observability
+  - id: edot-collector
+---
+
+# Kubernetes Cluster receiver
+
+The Kubernetes Cluster receiver (`k8s_cluster`) is a core component of the {{edot}} (EDOT) Collector. It collects cluster-level metrics and entity events from the Kubernetes API server, enabling observability into node health, resource allocation, and workload states in Elastic Observability.
+
+This receiver should be deployed as a single instance per cluster. It uses the Kubernetes API to watch for updates and emit metrics and events to your configured exporters.
+
+For full contrib details, see the [OpenTelemetry k8s_cluster receiver documentation](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/k8sclusterreceiver).
+
+
+## Get started
+
+To use the Kubernetes Cluster receiver, include it in the `receivers` section of your EDOT Collector configuration. The receiver is already bundled in the default EDOT Collector distribution.
+
+```yaml
+receivers:
+  k8s_cluster:
+    auth_type: serviceAccount
+```
+
+You can then reference it in your service pipelines to collect cluster-level metrics and entity events:
+
+```yaml
+service:
+  pipelines:
+    metrics:
+      receivers: [k8s_cluster]
+      exporters: [elasticsearch]
+```
+
+
+## How it works
+
+The receiver authenticates to the Kubernetes API (using a service account, kubeconfig, or no authentication) and continuously watches cluster objects such as Pods, Nodes, Deployments, and StatefulSets.
+
+It produces three kinds of data:
+
+- **Metrics** — cluster-wide and namespace-scoped performance data  
+- **Logs** — entity change events, when attached to a logs pipeline  
+- **Metadata updates** — sent to exporters that implement metadata sync interfaces
```

Comment:
Probably that can be skipped since it's not widely used and not really user-facing in EDOT.